src/SmsapiServiceProvider.php

```diff
@@ -44,7 +45,11 @@ public function boot()
                     });
                 }, $defaults);
 
-                return new SmsapiClient($client, $defaults);
+                $proxy = strtoupper($auth['service_domain']) == 'COM'
+                    ? new Native('https://api.smsapi.com')
```

Comment:
Actually, maybe just make this a constant, and set it as the default in the config file?

Reply:
I also thought about adding this as a constant but wasn't sure where. Also I'm not really happy with the "service_domain" naming. Would you introduce a new class like `SmsapiDomain::COM` and `SmsapiDomain::PL` or add it to an existing like `SmsapiClient::DOMAIN_COM` and `SmsapiClient::DOMAIN_PL`?

Reply:
Or just as a standalone constant `SMS_API_PL` and `SMS_PL_COM`?

Reply:
I would have gone `SmsapiClient::DOMAIN_COM` IMO

Reply:
Well, I went with `SmsapiClient::SERVICE_COM` and `auth.service` config. The new SMSAPI client uses a function instead of a proxy class (`$service = (new SmsapiHttpClient())->smsapiComService($apiToken);`) so this keeps this more open for future work.

Reply:
Perfect, thanks!